src/bin/menu_bar_system.rs

```diff
@@ -58,7 +58,7 @@ fn build_system_menu(application: &gtk::Application) {
     application.set_menubar(Some(&menu_bar));
 }
 
-// This function creates "actions" which connect on the declared actions from the menu items.
+/// This function creates "actions" which connect on the declared actions from the menu items.
```

Comment:
Why changing this to a doc comment?

Reply:
well, it is a comment on a function. And making it a doc comment enables it be shown in IDEs and the docs.
Other people may be interested in the function of this function.
So why not use the already existing comment :)